File: Datenbanksysteme/Solutions/homework1/hallo.hs
main :: IO () --TypDefinition
main = do -- main Funktion mit Kommentaren
    print ("Wie heisst du?") --console Ausgabe
    strName <- getLine --console String eingabe
    print ("Hallo " ++ strName ++ ".") --formatierte Ausgabe

{-Test in Konsole
habib@llangollen:~$ ./hallo
"Wie heisst du?"
Datenbanksystem 2017
"Hallo Datenbanksystem 2017."-}

File: Datenbanksysteme/Solutions/homework1/Hallo.java
//Programm mit Kommenaren
import java.util.Scanner;//Bibliothek importieren
public class Hallo {//Klasse Hallo
	public static void main(String[] args) {//main Methode
		System.out.println("Wie heisst du?");//Konsole Ausgabe
		Scanner scanner = new Scanner(System.in);//Scanner Objekt initialisieren
		String name = scanner.nextLine();//Konsole Eingabe
		scanner.close();//Scanner ordnungsgema:ss beenden
		System.out.println("Hallo " + name + ".");//formatierte Ausgabe
	}
}

/* Test in Konsole
Wie heisst du?
Datenbanksystem 2017
Hallo Datenbanksystem 2017.
*/